Image is missing dejavu-sans-fonts

I'm getting the following exception when I try to log in:

```
    Caused by: java.lang.NullPointerException
    	at sun.awt.FontConfiguration.getVersion(FontConfiguration.java:1264) [?:1.8.0_151]
    	at sun.awt.FontConfiguration.readFontConfigFile(FontConfiguration.java:219) [?:1.8.0_151]
    	at sun.awt.FontConfiguration.init(FontConfiguration.java:107) [?:1.8.0_151]
    	at sun.awt.X11FontManager.createFontConfiguration(X11FontManager.java:774) [?:1.8.0_151]
    	at sun.font.SunFontManager$2.run(SunFontManager.java:431) [?:1.8.0_151]
    	at java.security.AccessController.doPrivileged(Native Method) [?:1.8.0_151]
    	at sun.font.SunFontManager.<init>(SunFontManager.java:376) [?:1.8.0_151]
    	at sun.awt.FcFontManager.<init>(FcFontManager.java:35) [?:1.8.0_151]
    	at sun.awt.X11FontManager.<init>(X11FontManager.java:57) [?:1.8.0_151]
    	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method) [?:1.8.0_151]
    	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62) [?:1.8.0_151]
    	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) [?:1.8.0_151]
    	at java.lang.reflect.Constructor.newInstance(Constructor.java:423) [?:1.8.0_151]
    	at java.lang.Class.newInstance(Class.java:442) [?:1.8.0_151]
    	at sun.font.FontManagerFactory$1.run(FontManagerFactory.java:83) [?:1.8.0_151]
    	at java.security.AccessController.doPrivileged(Native Method) [?:1.8.0_151]
```
It seems to be caused by trying to display a captcha. 

The nearest I can find is https://jira.atlassian.com/browse/BSERV-6886, which mentions missing fonts. It wouldn't surprise me that the apline image is missing fonts compared to the debian image. I'd suggest the `ttf-dejavu` package is the one required on alpine.

--- a/Dockerfile
+++ b/Dockerfile
@@ -8,7 +8,7 @@ ENV JIRA_VERSION  7.8.0
 # Install Atlassian JIRA and helper tools and setup initial home
 # directory structure.
 RUN set -x \
-    && apk add --no-cache curl xmlstarlet bash \
+    && apk add --no-cache curl xmlstarlet bash ttf-dejavu \
     && mkdir -p                "${JIRA_HOME}" \
     && mkdir -p                "${JIRA_HOME}/caches/indexes" \
     && chmod -R 700            "${JIRA_HOME}" \

